libkbfs/bserver_remote.go

```diff
@@ -25,41 +26,147 @@ const (
 	BServerTokenServer = "kbfs_block"
 	// BServerTokenExpireIn is the TTL to use when constructing an authentication token.
 	BServerTokenExpireIn = 2 * 60 * 60 // 2 hours
+	// BServerDefaultPingIntervalSeconds is the default interval on which the
+	// client should contact the block server.
+	BServerDefaultPingIntervalSeconds = 10
+	// BServerPingTimeout is how long to wait for a ping response
+	// before breaking the connection and trying to reconnect.
+	BServerPingTimeout = 30 * time.Second
```

Comment:
The pings are launched from a single goroutine that blocks on each ping, so there should only ever be one outstanding as far as I can tell.  I was worried if I made it too small, I'd end up missing pings when uploading lots of data, or when the server is busy, and in that case re-initializing the connection would just make the problem worse.

Not sure what the right balance is.  Any thoughts?